features/step_definitions/test_steps.rb

```diff
@@ -204,7 +204,29 @@
   pending
 end
 
-Then(/^I select "([^"]*)" from the dropdown list with classname "([^"]*)"$/) do |name,classname|
-  browser.div(:class,classname).click
-  browser.select_list(:class => classname).div(:text => name).select
+
+Then(/^I click on the drop down box with div class "([^"]*)"$/) do |dropdown|
+  browser.div(:class,dropdown).click
+end
+
+
+Then(/^I select "([^"]*)" with index "([^"]*)" with div class "([^"]*)"$/) do |name, index, classname|
+
+end
+
+
+Then(/^I print the contents of dropdown box with the class name "([^"]*)"$/) do |classname|
+  browser.divs(:class => classname).each do |div|
+    puts div.text
+  end
+end
+
```

Comment:
@divhegde Agreed, Both are same! but, this step, I introduced it, to verify, as to what `class` is holding those contents in the drop box, so that the valid class would print the contents. Now that, the class is identified, once the element in the class is able to be clicked, I ll remove this step, and proceed to modifying step definitions in the scenarios you have written. Hence, not merging this.